akvo/rsr/views/py_reports.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+"""Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the
+Akvo RSR module. For additional details on the GNU license please
+see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+import os
+import pdfkit
+from django.http import HttpResponse
+from django.template.loader import render_to_string
+
+
+def check(request):
+    is_reports_container = os.getenv('DJANGO_PORT', None) == '9000'
```

Comment:
Use a specific env variable "IS_REPORT_CONTAINER". 

It is going to be very confusing if we change the port number and then suddenly reports stop working.